code/datums/dna.dm

```diff
@@ -510,6 +510,8 @@
 			var/datum/mutation/human/HM = dna.get_mutation(mutation)
 			if(HM)
 				HM.scrambled = TRUE
+				if(resilient && HM.quality!=noperma) //note that noperma only works for single categories; feeding, say, NEGATIVE + MINOR_NEGATIVE in as the noperma parameter will make the noperma parameter not do anything 

```

Comment:
It would if the quality was NEGATIVE + MINOR_NEGATIVE

use the bit operators `|` and `&`

Reply:
In what way, exactly?

Reply:
`HM.quality & (NEGATIVE | MINOR_NEGATIVE)` I think

Reply:
Okay, I'm trusting you on this. I had remembered a bad bug with coughing's airborne transmission threshold that involved an &, so I was hesitant to use them.

Reply:
@ExcessiveUseOfCobblestone Wait, wouldn't doing this your way remove the NEGATIVE and/or MINOR_NEGATIVE flags from the added mutation?

Reply:
If you can absolutely ASSURE me that doing it your way won't break stuff, I'll do it your way. If we do this wrong, though, we could be planting the seeds for a bug or two down the line that it would be really hard to identify the cause(s) of.

Reply:
boo hiss

Reply:
do the change